docs/remote-access/wireguard.md

```diff
@@ -11,4 +11,112 @@ This page will cover two WireGuard architectures. The first is running WireGuard
 
 ## WireGuard Server on a VPS
 
-* Todo
\ No newline at end of file
+This section will talk about setting up a VPS and then running a client on your network to setup the connection to the VPS and allow the vpn to work without a firewall opening. 
+
+The first step is to decide on a virtual host provider and create a server. When you decide on a provider and create a host. Install Wireguard using the instructions provided on the [Wireguards Documentation](https://www.wireguard.com/install/)
```

Comment:
> The first step is to decide on a virtual host provider

@IronicBadger would it be out of place to put an SSH affiliate code here?

Reply:
Maybe. I'm not sure. I'll think on it.

Reply:
Is there another location on this site that we could link to instead. That way the guide itself is generic, then you could include a affiliate codes if you wanted or just links out to the provider. So there would only be an update in one location instead of through out the Wiki.